targets/csharp/source/PlayFabSDK/source/PlayFabHttp/PlayFabPollyHttp.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Xbox.Publishing.PlayFabCSharpSDK.Polly.Interfaces;
+using PlayFab;
+using PlayFab.Internal;
+using Polly;
+using Polly.Wrap;
+
+namespace PlayFab.Internal
+{
+    /// <summary>
+    /// A Polly wrapped version of PlayFab Transport plug in which used
+    /// when making http requests to PlayFab Main Server.
+    /// </summary>
+    public class PlayFabPollyHttp : PlayFabHttp, IPlayFabPlugin
```

Comment:
I think you want to inherit from ITransportPlugin, not IPlayFabPlugin. That should ensure this plugin can be used interchangably with the existing PlayFabSysHttp transport.